k8s/eks/README.md

```diff
@@ -0,0 +1,284 @@
+This guide will help you setup an EKS (Elastic Kubernetes Service) cluster, and run testground plans on it.
+
+## Requirements
+
+It is assumed that you already have the following:
+
+- A bastion host on AWS, or a laptop with AWS credentials which will be used to run the installation script, interact with the cluster, and run testground plans
+- The laptop or bastion host has the following software installed:
+
+**Note: the software and installation guides have been tested on amazon linux. Please verify which OS are you using and follow the official guides accordingly.
+Links have been included for each software requirement. Listed guides are for example purposes only.**
+
+**1. docker**
+
+Official installation guide:
+
+https://docs.docker.com/engine/install/ubuntu/
+
+Installation on amazon linux:
+
+```
+sudo yum install docker -y
+sudo systemctl enable docker.service
+sudo systemctl start docker.service
+sudo usermod -aG docker ec2-user
+newgrp docker
+```
```

Comment:
While I agree with you, these comments were added with the main purpose of aiding you with first time setup and not wasting time searching for correct commands as the official docs are sometimes not updated. 
These are actually tested commands and could save you time for initial testing.

Reply:
I'd think docker official docs would be getting more frequent updates than the testground ones. That's why I'd like to see links to official docs instead of specific instructions that work on your machine.

Reply:
And that is why we have included both links to the offical docs as first lines for each dependency, and added these notes to make it easier for you or any other user. 
Please read the README file again.